FestivalPlanner/theaters/tests.py

```diff
@@ -1,3 +1,114 @@
-from django.test import TestCase
+from http import HTTPStatus
 
-# Create your tests here.
+from django.urls import reverse
+
+from festivals.tests import create_festival, mock_base_festival_mnemonic
+from films.tests import ViewsTestCase
+from theaters.models import City, Theater, Screen
+
+
+class TheaterViewTests(ViewsTestCase):
+
+    def setUp(self):
+        super().setUp()
+
+        # Set up a festival. Needed to render any page in the app.
+        self.city = City.cities.create(city_id=1, name='Berlin', country='de')
+        self.festival = self.create_festival(self.city, '2023-02-16', '2023-02-26')
+
+        # Create some URL shorthands.
+        self.theaters_url = reverse('theaters:theaters')
+        self.details_url = lambda pk: reverse('theaters:details', args=[pk])
+
+    def tearDown(self):
+        super().tearDown()
+
+    @staticmethod
+    def create_festival(city, start_data_str, end_date_str):
+        return create_festival(mock_base_festival_mnemonic(), city, start_data_str, end_date_str)
```

Comment:
Please explain why a festival is needed.

Reply:
It's explained as a comment above the call of this method.
All pages need a festival (when logged in).